CHANGELOG.md

```diff
@@ -18,12 +18,70 @@ The release notes for the ROCm platform.
 ## ROCm 5.6.0
 <!-- markdownlint-disable first-line-h1 -->
 <!-- markdownlint-disable no-duplicate-header -->
+### What's New in This Release
```

Comment:
This section is not needed

Reply:
Delete to line 27

Reply:
Do we want to do the same for the Release Notes (`RELEASE.md`) or just the change log?

Reply:
Thanks, good point. Done in [38f26e4](https://github.com/RadeonOpenCompute/ROCm/pull/2259/commits/38f26e4da29adadd0b9ab04030001c74b77ff8b5)